arbeitszeit_web/presenters/company_purchases_presenter.py

```diff
@@ -30,13 +33,19 @@ def append(self, purchase_respond: PurchaseQueryResponse) -> None:
             ),
             product_name=purchase_respond.product_name,
             product_description=purchase_respond.product_description,
-            purpose=purchase_respond.purpose,
+            purpose=self._convert_purpose(purchase_respond.purpose),
             price_per_unit=str(round(purchase_respond.price_per_unit, 2)),
             amount=str(purchase_respond.amount),
             price_total=str(round(purchase_respond.price_total, 2)),
         )
         self.purchases.append(p)
 
+    def _convert_purpose(self, purpose: PurposesOfPurchases) -> str:
```

Comment:
I think that "convert" is to general here and would prefer if this method was named "_format_pupose"